<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crear</class>
 <widget class="QDialog" name="crear">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagenes/images.qrc">
    <normaloff>:/images/calendar.png</normaloff>:/images/calendar.png</iconset>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0.391089 rgba(243, 243, 243, 133), stop:0.915842 rgba(165, 157, 153, 133));</string>
   </property>
   <property name="text">
    <string>Nueva tarea :)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>401</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0.391089 rgba(243, 243, 243, 133), stop:0.915842 rgba(165, 157, 153, 133));</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Tarea:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textline_Tarea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ingresa el nombre d ela tarea</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Dia:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Dia">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Dia de la semana</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lunes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Martes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Miercoles</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jueves</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Viernes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sabado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Domingo</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Prioridad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Hora:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit_Hora">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Descripci√≥n</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/addEvent.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Descripcion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>331</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Prioridad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>--</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Alta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Media</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Baja</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_Guardar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="icon">
    <iconset resource="imagenes/images.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imagenes/images.qrc"/>
 </resources>
 <connections/>
</ui>
